app/scripts/modules/core/application/modal/pageApplicationOwner.html

```diff
@@ -0,0 +1,31 @@
+<div modal-page>
+  <task-monitor monitor="taskMonitor"></task-monitor>
+  <modal-close></modal-close>
+  <div class="modal-header">
+    <h3 data-purpose="modal-header">Page {{application.name}} Owner</h3>
+  </div>
+
+  <form role="form" class="container-fluid" novalidate name="pageApplicationOwnerForm">
+    <div class="modal-body">
+
+      <div class="row" style="margin-top:10px; margin-bottom:10px">
```

Comment:
This is effectively the same as the `reason` directive safe for different placeholder text.